next.config.js

```diff
@@ -1,16 +1,22 @@
 /** @type {import('next').NextConfig} */
 
 const nextConfig = {
-  basePath: process.env.NODE_ENV === 'production' ? '/openbc-web' : '',
-  output: 'export',
-  distDir: 'dist',
+  basePath: process.env.NODE_ENV === 'production' ? '' : '',
+  output: 'standalone',
   githubApiToken: process.env.GITHUB_API_KEY,
   images: {
     unoptimized: true,
   },
   typescript: {
     tsconfigPath: './tsconfig.json',
   },
+  webpackDevMiddleware: (config) => {
```

Comment:
Are we sure we need this?

I get a warning that this is not recognized.

@LeoQiu981112 , @alchuu00 , @umsu2

Reply:
this seems like it's used for dev watching purposes. not sure how this will work in production. 
working with docker + watching the files is not trivial. what happens with file mounting is that the the node runtime in your local system is doing the transpiling, then gets mounted over to the docker node runtime which runs the code. 
what we probably want is to have the build/run command run from the docker container instead of your own system. the best litmus test to have it setup properly is to actually not have node.js installed on your machine. can you use docker, docker compose to run the app in container. that's the only way to garantee consistency in terms of build across dev machines/production

Reply:
Trying to hot reload and having it mimic product might be to much work right now.

Perhaps we could forgo hot reload for now and just focus on building it like production.

Create a ticket to look into hot reload.